Updated lists of SUSE Linux Enterprise hardening profiles in openSCAP (#3114)
uyuni-project/uyuni-docs: modules/administration/pages/openscap.adoc

[IMPORTANT]
====
{suse} provides the [package]``scap-security-guide`` package for different openscap profiles.

> In the current version of [package]``scap-security-guide``, {suse} supports the following profiles for {sles} 15:

Reviewer: Keep the existing format and use the following list

- DISA STIG profile for SUSE Linux Enterprise Server 12 and 15
- ANSSI-BP-028 profile for SUSE Linux Enterprise Server 12 and 15
- PCI-DSS profile for SUSE Linux Enterprise Server 12 and 15
- HIPAA profile for SUSE Linux Enterprise Server 12 and 15
- CIS profile for SUSE Linux Enterprise Server 12 and 15
- Hardening for Public Cloud Image of SUSE Linux Enterprise Server for SAP Applications 15
- Public Cloud Hardening for SUSE Linux Enterprise 15

*  CIS SUSE SUSE Linux Enterprise 15 Benchmark Level 2 (Workstation)
*  CIS SUSE Linux Enterprise 15 Benchmark for Level 1 (Server)
*  CIS SUSE Linux Enterprise 15 Benchmark for Level 1 (Workstation)
*  CIS SUSE Linux Enterprise 15 Benchmark for Level 2 (Server)
*  DISA STIG for SUSE Linux Enterprise 15
*  Hardening for Public Cloud Image of SUSE Linux Enterprise Server (SLES) for SAP Applications 15
*  Health Insurance Portability and Accountability Act (HIPAA)
*  PCI-DSS v4 Control Baseline for SUSE Linux Enterprise 15
*  Public Cloud Hardening for SUSE Linux Enterprise 15
*  Standard System Security Profile for SUSE Linux Enterprise 15

For {sles} 12:

* ANSSI-BP-028 (enhanced)
* ANSSI-BP-028 (high)
* ANSSI-BP-028 (intermediary)
* ANSSI-BP-028 (minimal)
* CIS SUSE SUSE Linux Enterprise 12 Benchmark Level 2 (Workstation)
* CIS SUSE Linux Enterprise 12 Benchmark for Level 1 (Server)
* CIS SUSE Linux Enterprise 12 Benchmark for Level 1 (Workstation)
* CIS SUSE Linux Enterprise 12 Benchmark for Level 2 (Server)
* DISA STIG for SUSE Linux Enterprise 12
* PCI-DSS v.4 Control Baseline for SUSE Linux Enterprise 12
* Standard System Security Profile for SUSE Linux Enterprise 12

> // Other profiles, like the CIS profile, are community supplied and not officially supported by {suse}.

Reviewer: ```suggestion
// Any other profiles are community supplied and not officially supported by {suse}.
```

In the current version of [package]``scap-security-guide``, {suse} supports the following profiles:

* DISA STIG profile for {sles} 12 and 15
* ANSSI-BP-028 profile for {sles} 12 and 15
* PCI-DSS profile for {sles} 12 and 15

> * HIPAA profile for {sles} 12 and 15

Reviewer 2: HIPPA is available only for SLES 15, and not SLES12.

* CIS profile for {sles} 12 and 15
* Hardening for Public Cloud Image of {sles} for SAP Applications 15
* Public Cloud Hardening for {sle} 15

> 

Reviewer 2: Standard System Security profile for SLES 15 and 12 is missing from the list.

References (see top right drop-down in the following pages):
- https://static.open-scap.org/ssg-guides/ssg-sle15-guide-index.html
- https://static.open-scap.org/ssg-guides/ssg-sle12-guide-index.html
